app/src/renderer/connectors/node.js

```diff
@@ -1,11 +1,49 @@
 "use strict"
 
-const axios = require(`axios`)
+const { ipcRenderer } = require(`electron`)
 const RestClient = require(`./lcdClient.js`)
 const mockedRestClient = require(`./lcdClientMock.js`)
 const RpcWrapper = require(`./rpcWrapper.js`)
 const MockedRpcWrapper = require(`./rpcWrapperMock.js`)
 
+// Proxy requests to Axios through the main process because we need
+// Node.js in order to support self-signed TLS certificates.
+const AxiosProxy = () => {
+  let requestCounter = 0
+
+  return options =>
+    new Promise((resolve, reject) => {
+      requestCounter++
+
+      if (requestCounter === Number.MAX_SAFE_INTEGER) {
+        requestCounter = 0
+      }
+
+      const channel = `Axios/${requestCounter}`
+
+      console.log(`Request ${options.method.toUpperCase()} ${options.url}`)
```

Comment:
Delete console log as we already have it for both cases 
```suggestion
```

Reply:
This is the "Request sent" message the others or "success" and "failure"